<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BilleteraApp</class>
 <widget class="QDialog" name="BilleteraApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0.207, y2:0.954545, stop:0.132159 rgba(156, 178, 178, 255), stop:1 rgba(255, 255, 255, 255))}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:600; color:#ffffff;&quot;&gt;BilleterApp&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="quantity_line">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>160</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:5px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Quantity</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="category_line">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>230</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:5px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Category</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="subcategory_line">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>300</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:5px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Subcategory</string>
    </property>
   </widget>
   <widget class="QLabel" name="quantity_label">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>140</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantity</string>
    </property>
   </widget>
   <widget class="QLabel" name="category_label">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>210</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Category</string>
    </property>
   </widget>
   <widget class="QLabel" name="subcategory_label">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>280</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Subcategory</string>
    </property>
   </widget>
   <widget class="QLabel" name="description_label">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>350</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_button">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>460</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QComboBox" name="accounts_comboBox">
    <property name="geometry">
     <rect>
      <x>347</x>
      <y>160</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="account_label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>140</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Account</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="description_line">
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>370</y>
      <width>501</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:5px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Description</string>
    </property>
   </widget>
   <widget class="QLabel" name="total_label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>250</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Total: </string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>accounts_comboBox</tabstop>
  <tabstop>quantity_line</tabstop>
  <tabstop>category_line</tabstop>
  <tabstop>subcategory_line</tabstop>
  <tabstop>description_line</tabstop>
  <tabstop>save_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
